site_scons/components/__init__.py

```diff
@@ -132,20 +111,12 @@ def define_mercury(reqs):
                 commands=['./autogen.sh',
                           './configure --prefix=$OFI_PREFIX ' +
                           '--disable-efa ' +
+                          '--disable-psm3 ' +
                           '--without-gdrcopy ' +
-                          OFI_DEBUG +
-                          exclude(reqs, 'psm2',
-                                  '--enable-psm2' +
-                                  check(reqs, 'psm2',
-                                        "=$PSM2_PREFIX "
-                                        'LDFLAGS="-Wl,--enable-new-dtags ' +
-                                        '-Wl,-rpath=$PSM2_PREFIX/lib64" ',
-                                        ''),
-                                  ''),
```

Comment:
Why don't we explicitly disable it? Just like the other providers
`--disable-psm2`

Reply:
If it is decided to remove psm2 support. You may consider removing psm2 from the scons building EXCLUDE list
https://github.com/daos-stack/daos/blob/0e8cb15cd627441f8e16cf12c267869cb934d3fb/site_scons/prereq_tools/base.py#L683

Reply:
If it is decided to remove psm2 support. You may consider removing psm2 from the scons building EXCLUDE list
https://github.com/daos-stack/daos/blob/0e8cb15cd627441f8e16cf12c267869cb934d3fb/site_scons/prereq_tools/base.py#L683

Reply:
Yes, agreed.  I had some comments with Jeff about this before he went out on holiday.

Given the way this is going I'm leaning back more towards having a PR just to strip out the PSM2 code.  The reason the change to disable it is in here is that it doesn't build on the newer Ubuntu versions.

Reply:
I created #5927 to address [DAOS-7849](https://jira.hpdd.intel.com/browse/DAOS-7849)